InvenTree/InvenTree/tasks.py

```diff
@@ -306,4 +307,35 @@ def send_email(subject, body, recipients, from_email=None):
         from_email,
         recipients,
         fail_silently=False,
+        html_message=html_message
     )
+
+
+def notify_low_stock(stock_item):
+    """
+    Notify users who have starred a part when its stock quantity falls below the minimum threshold
+    """
+
+    from allauth.account.models import EmailAddress
+    starred_users = EmailAddress.objects.filter(user__starred_parts__part=stock_item.part)
+
+    if len(starred_users) > 0:
+        logger.info(f"Notify users regarding low stock of {stock_item.part.name}")
+        body = f'Hi, {stock_item.part.name} is low on stock. Kindly do the needful.'
```

Comment:
This needs to be translated

Reply:
@rocheparadox I would prefer if we used a template file for this as well - makes it easier for users to override the messages without messing with functional code.